staging/src/k8s.io/apiserver/pkg/util/flowcontrol/apf_controller.go

```diff
@@ -177,6 +177,9 @@ type configController struct {
 	// name to the state for that level.  Every name referenced from a
 	// member of `flowSchemas` has an entry here.
 	priorityLevelStates map[string]*priorityLevelState
+
+	// nominalCLSum is the sum of the nominalCL fields in the priorityLevelState records
+	nominalCLSum int
```

Comment:
See https://github.com/kubernetes/enhancements/pull/3643 for why the server concurrency limit is not necessarily the required sum going into the concurrency solver.

Reply:
Latest revision expands the commentary here.